### PR TITLE
Create popover tokens

### PR DESCRIPTION
**Pull request by SaiSan-WMDE**
This makes available the component tokens to be used to style WiKit's popover component. Very early version, WIP, needs review.

Ticket: [T273041](https://phabricator.wikimedia.org/T273041)

### CODE REVIEW

File: tokens/properties/components/Popover.json [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+                "comment": "Show and hide delays"
+            }
+        },
+        "distance": {
+            "value": "{dimension.spacing.xsmall.value}",
+            "comment": "this reffers to the space between the trigger and the popover arrow. Better names welcome"
+        },
+        "pointer":  {
+            "width": {
+                "value": "{dimension.spacing.xxlarge.value}"
```
Comment (SaiSan-WMDE) [inline tokens/properties/components/Popover.json:54]:
```suggestion
                "value": "{dimension.spacing.large.value}"
```

File: tokens/properties/components/Popover.json [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        "box-shadow": {
+            "value": "{box-shadow.elevation.dialog.value}"
+        },
+        "transition": {
+            "delay": {
+                "value":"{transition.duration.fast.value}",
+                "comment": "Show and hide delays"
+            }
+        },
+        "distance": {
```
Comment (SaiSan-WMDE) [inline tokens/properties/components/Popover.json:48]:
```suggestion
        "gap": {
```

File: tokens/properties/components/Popover.json [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+        },
+        "transition": {
+            "delay": {
+                "value":"{transition.duration.fast.value}",
+                "comment": "Show and hide delays"
+            }
+        },
+        "gap": {
+            "value": "{dimension.spacing.xsmall.value}",
+            "comment": "this reffers to the space between the trigger and the popover arrow. Better names welcome"
```
Comment (SaiSan-WMDE) [inline tokens/properties/components/Popover.json:50]:
```suggestion
            "comment": "this reffers to the space between the target and the popover pointer. Better names welcome"
```

Comment (SaiSan-WMDE) [inline tokens/properties/components/Popover.json]:
```suggestion
            "comment": "Aka "distance". This reffers to the space between the trigger and the popover arrow. Better names welcome"
```

File: tokens/properties/components/Popover.json [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+        },
+        "transition": {
+            "delay": {
+                "value":"{transition.duration.fast.value}",
+                "comment": "Show and hide delays"
+            }
+        },
+        "gap": {
+            "value": "{dimension.spacing.xsmall.value}",
+            "comment": "this reffers to the space between the target and the popover pointer. Better names welcome"
```
Comment (SaiSan-WMDE) [inline tokens/properties/components/Popover.json:50]:
```suggestion
            "comment": "Aka distance. This reffers to the space between the target and the popover pointer. Better names welcome"
```

File: tokens/properties/components/Popover.json [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+        },
+        "transition": {
+            "delay": {
+                "value":"{transition.duration.fast.value}",
+                "comment": "Show and hide delays"
+            }
+        },
+        "gap": {
+            "value": "{dimension.spacing.xsmall.value}",
+            "comment": "Aka distance. This reffers to the space between the target and the popover pointer. Better names welcome"
```
Comment (SaiSan-WMDE) [inline tokens/properties/components/Popover.json:50]:
```suggestion
            "comment": "Aka distance. This refers to the space between the target and the popover pointer. Better names welcome"
```

File: tokens/properties/components/Popover.json [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+            "value": "{dimension.spacing.medium.value}"
+        },
+        "border-style": {
+            "value": "{border.style.base.value}"
+        },
+        "border-width": {
+            "value": "{border.width.thin.value}"
+        },
+        "border-color": {
+            "value": "{border.color.base.subtle.value}"
```
Comment (SaiSan-WMDE) [inline tokens/properties/components/Popover.json:28]:
```suggestion
            "value": "{border.color.base.default.value}"
```

Comment (SaiSan-WMDE) [reply tokens/properties/components/Popover.json:28]: updated based on WMF's designs

**Review by Ladsgroup — APPROVED** [review]
Approving the tokens

**Pull request merged**